versions/1/manifest.spec.yml

```diff
@@ -202,10 +198,69 @@ spec:
                 description:
                   description: Longer description of input.
                   type: string
-                  examples: 
+                  examples:
                   - Collecting Apache access and error logs
                 vars:
-                  $ref: "./dataset/manifest.spec.yml#streams/items/properties/vars"
+                  description: Input variables.
```

Comment:
Aren't we duplicating this spec? That's why I had the `$ref` originally.

Reply:
Yes, but the provided loader doesn't support the statik filesystem. I can try to implement a custom resource loader.

Also, the library enforces a canonical path here with schema, so it would be: `file://./dataset/manifest.spec.yml#streams/items/properties/vars`

Reply:
I see. I think the duplication is fine for now but we should probably make comments in both places pointing to the other as a duplicate that needs to be kept in sync whenever any one of the places is changed. Longer term, let's look into implementing a custom resource loader.

Reply:
I will try to focus on this as followup. Got interested with this :)